presto-docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -0,0 +1,173 @@
+=================
+Phoenix Connector
+=================
+
+The Phoenix connector allows querying data stored in HBase using Apache Phoenix.
+
+Compatibility
+-------------
+
+The Phoenix connector is compatible with all Phoenix versions starting from 4.14.x.
+
+Configuration
+-------------
+
+To configure the Phoenix connector, create a catalog properties file
+``etc/catalog/phoenix.properties`` with the following contents,
+replacing ``host1,host2,host3`` with a comma-separated list of the ZooKeeper
+nodes used for discovery of the HBase cluster:
+
+.. code-block:: none
+
+    connector.name=phoenix
+    phoenix.connection-url=jdbc:phoenix:host1,host2,host3:2181:/hbase
+    phoenix.hbase-site-path=/path/to/hbase-site.xml
```

Comment:
for hive we are using `hive.config.resources` and this accepts a list of hadoop configuration files. Maybe here we could do similar `phoenix.config.resources`?